### PR TITLE
without check ships position

### PR DESCRIPTION
**Pull request by bythewaters**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,34 +1,133 @@
+class NumberShips(Exception):
+    pass
+
+
+class FleetSize(Exception):
+    pass
```
Comment (Abnormaltype) [inline app/main.py:6]: use correct naming for `Errors`

File: app/main.py [hunk 2]
```diff
@@ -1,34 +1,133 @@
...
+        for row in self.battle_map():
+            print((" " * 6).join(map(str, row)))
+
+    def fire(self, location: tuple) -> str:
+        ship = self.fields.get(location, None)
+        if ship is not None:
+            ship.fire(location[0], location[1])
+            if ship.is_drowned:
+                return "Sunk!"
+            else:
```
Comment (Abnormaltype) [inline app/main.py:131]: unnecessary else

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,34 +1,132 @@
+class NumberOfShipsDeck(Exception):
```
Comment (Abnormaltype) [inline app/main.py:1]:
```suggestion
class NumberOfShipsDeckError(Exception):
```

File: app/main.py [hunk 4]
```diff
@@ -1,34 +1,132 @@
...
+    pass
+
+
+class TotalNumberOfTheShips(Exception):
```
Comment (Abnormaltype) [inline app/main.py:5]:
```suggestion
class TotalNumberOfTheShipsError(Exception):
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by YuriiKindrat — APPROVED** [review]

**Pull request closed without merging**